lib/foods.js

```diff
@@ -14,13 +14,66 @@ function checkInputs() {
   if ( $('#name-field input').val() != '' && $('#calories-field input').val() != '') {
     var food = $('#name-field input').val();
     var calories = $('#calories-field input').val();
+    var foodsRaw = sessionStorage.getItem("foods");
+    var foods = $.parseJSON(foodsRaw);
+    // if (foods == null) {
+    //   foods = [];
+    // };
+    // foods.push(createNewFood(food, calories));
+    // sessionStorage.setItem("foods", JSON.stringify(foods));
 
-    $('tbody').prepend('<tr class="food-row"><td>' + food + '</td><td class="food-calories">' + calories + '</td><td class="food-delete"><button>-</button></td></tr>');
+    // $('tbody').prepend('<tr class="food-row"><td class="food-name">' + food + '</td><td class="food-calories">' + calories + '</td><td class="food-delete"><button>-</button></td></tr>');
+    foodTableSingleRow(food, calories, true, false, true)
+    saveFoodList();
     $('#name-field input').val("");
     $('#calories-field input').val("");
   };
 };
 
+function foodTableSingleRow(name, calories, deleteButton, checkBox, addToList){
```

Comment:
@lucyconklin For some reason your comment about this line does not show up here, but I received an email notification about it... :-) For reference here is the comment: "I'm guessing this is how we get those checkboxes, and create those two tables at the same time."
The answer is yes, at least the function `foodTableSingleRow` is already prepared with parameters we can later use to decide whether we need the table row built with `delete` button or `checkbox`. It would be flexible enough to be able to build it with both or with none, that should come handy when we would build the `meal` tables. Single responsibility! :-)